examples/analog_hamiltonian_simulation/06_Analog_Hamiltonian_simulation_with_PennyLane.ipynb

```diff
@@ -22,7 +22,9 @@
    "cell_type": "code",
```

Comment:
<p>Isn't this too expensive to run this example? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Also, what do you mean by no cost to run this notebook? But I see the cost tracker computing the amount?

Reply:
1. I am rerunning the notebook with 100 shots and see if the results make sense. 
2. The cost to run the notebook is not shown previously, despite the cost track is there.

Reply:
<p>It turns out that shots=100 can already give the sensible result [a bit surprise given that we have to drop some results with nan]. With that, we have reduced the cost from $20 to $2!</p>